addons/expanded-backpack/addon.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "Expanded backpack",
```

Comment:
Please rename this addon.

Reply:
What name were you suggesting? Extended backpack, Better backpack?

Reply:
I'd personally say "Advanced Backpack" or "Expandable Backpack".

Reply:
Don't use an adjective. Describe the functionality.

Reply:
What about "Larger Backpack" or "Increased Backpack Size"?

Reply:
then again that might confuse users into thinking that it increases the amount of items you can store in the backpack, rather than making it easier to find stuff in it
it has other features like improving the look of the costume and script icons which don't really fit the title description
i chose that name because its a general addon that "expands" the functionality of the backpack by adding a few different useful features but am not exactly sure what we should call it

was also planning on having basic sort/filter buttons above the items inside the backpack (like the costume library) to show only costumes, sounds, scripts or sprites and not the others, but that would require quite a bit of javascript to show/hide each item depending on what kind it is and that im not familiar enough with js to write such a thing yet